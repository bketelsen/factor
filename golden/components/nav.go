package components

import (
	"github.com/factorapp/factor/markup"
)

/*
type Nav struct {
	MyProp      string
	CurrentPath string
}
*/

func (n *Nav) OnAfterPrint(e *markup.Event) {

}
