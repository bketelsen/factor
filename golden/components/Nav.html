<nav>
    <ul>
        <li>This a component property: {vecty-field:MyProp}</li>
        <li>This is another item</li>
    </ul>
</nav>
